src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -81,10 +86,10 @@ public static ToGerritRunListener getInstance() {
     }
 
     @Override
-    public synchronized void onCompleted(AbstractBuild r, TaskListener listener) {
+    public synchronized void onCompleted(AbstractBuild r, @Nonnull TaskListener listener) {
         GerritCause cause = getCause(r);
         logger.debug("Completed. Build: {} Cause: {}", r, cause);
-        if (cause != null) {
+        if (r != null && cause != null) {
```

Comment:
Minimal effort to get rid of the warning that r might be null.
The method in RunListener is declared as ```public void onCompleted(R r, @Nonnull TaskListener listener) {}```